### DISCUSSION TITLE
Search for narrator

### DISCUSSION THREAD

**Discussion by Travillion** [advplyr/audiobookshelf#1494] [Ideas]
Would be helpful if tagged narrators were a searchable entity. For example, looking for titles narrated by Scott Brick. All of my audiobooks are fully tagged with embedded metadata, which ABS uses as preferred source. ABS correctly recognizes my narrators, but searching for a narrator doesn't produce any results.

**Comment by advplyr** [comment]
Created feature request https://github.com/advplyr/audiobookshelf/issues/1495
